libs/ob-sql-parser/src/main/resources/obmysql/sql/OBParser.g4

```diff
@@ -2318,6 +2516,11 @@ relation_sep_option
     : Comma?
     ;
 
+relation_factor_in_mv_hint_list
```

Comment:
this expression is never used, we may delete it

Reply:
done